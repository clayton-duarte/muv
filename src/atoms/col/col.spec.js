import React from 'react';

import Col from '.';

it('should render a col', () => {
  const wrapper = mount(
    <Col />,
  );
  expect(wrapper).toMatchSnapshot();
});
