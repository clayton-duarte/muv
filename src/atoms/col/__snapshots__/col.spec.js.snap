// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a col 1`] = `
<Col
  align="flex-start"
  grow={1}
  justify="flex-start"
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "error": "#dd3333",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".875rem 1rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        "size": "1rem",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".3s ease",
      },
    }
  }
>
  <Styled(Row)
    align="flex-start"
    grow={1}
    justify="flex-start"
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "error": "#dd3333",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".875rem 1rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "1rem",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <Row
      align="flex-start"
      className="sc-htpNat cPqbHx"
      grow={1}
      justify="flex-start"
      theme={
        Object {
          "color": Object {
            "black": "#484848",
            "error": "#dd3333",
            "gray": "#f0f0f0",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0082c8",
            "secondary": "#00baf2",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".875rem 1rem",
            "radius": ".25rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "1rem",
          },
          "text": Object {
            "color": "#484848",
            "size": "1rem",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <Styled(Container)
        align="flex-start"
        className="sc-htpNat cPqbHx"
        grow={1}
        justify="flex-start"
        theme={
          Object {
            "color": Object {
              "black": "#484848",
              "error": "#dd3333",
              "gray": "#f0f0f0",
              "overlay": "rgba(0, 0, 0, .15)",
              "primary": "#0082c8",
              "secondary": "#00baf2",
              "success": "#33bb33",
              "warn": "#dd3333",
              "white": "#fefefe",
            },
            "shape": Object {
              "border": ".125rem solid rgba(0, 0, 0, .15)",
              "margin": ".25rem .125rem",
              "padding": ".875rem 1rem",
              "radius": ".25rem",
              "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
              "size": "1rem",
            },
            "text": Object {
              "color": "#484848",
              "size": "1rem",
            },
            "transition": Object {
              "time": ".3s ease",
            },
          }
        }
      >
        <Container
          align="flex-start"
          className="sc-htpNat cPqbHx sc-bwzfXH kHNFvR"
          full={false}
          grow={1}
          height="auto"
          justify="flex-start"
          maxHeight="auto"
          maxWidth="auto"
          minHeight="auto"
          minWidth="auto"
          theme={
            Object {
              "color": Object {
                "black": "#484848",
                "error": "#dd3333",
                "gray": "#f0f0f0",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0082c8",
                "secondary": "#00baf2",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "padding": ".875rem 1rem",
                "radius": ".25rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "1rem",
              },
              "text": Object {
                "color": "#484848",
                "size": "1rem",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            }
          }
          width="auto"
        >
          <styled.div
            align="flex-start"
            className="sc-htpNat cPqbHx sc-bwzfXH kHNFvR"
            full={false}
            grow={1}
            height="auto"
            justify="flex-start"
            maxHeight="auto"
            maxWidth="auto"
            minHeight="auto"
            minWidth="auto"
            theme={
              Object {
                "color": Object {
                  "black": "#484848",
                  "error": "#dd3333",
                  "gray": "#f0f0f0",
                  "overlay": "rgba(0, 0, 0, .15)",
                  "primary": "#0082c8",
                  "secondary": "#00baf2",
                  "success": "#33bb33",
                  "warn": "#dd3333",
                  "white": "#fefefe",
                },
                "shape": Object {
                  "border": ".125rem solid rgba(0, 0, 0, .15)",
                  "margin": ".25rem .125rem",
                  "padding": ".875rem 1rem",
                  "radius": ".25rem",
                  "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                  "size": "1rem",
                },
                "text": Object {
                  "color": "#484848",
                  "size": "1rem",
                },
                "transition": Object {
                  "time": ".3s ease",
                },
              }
            }
            width="auto"
          >
            <div
              className="sc-htpNat cPqbHx sc-bwzfXH kHNFvR sc-bdVaJa eGlEDl"
              height="auto"
              width="auto"
            >
              col
            </div>
          </styled.div>
        </Container>
      </Styled(Container)>
    </Row>
  </Styled(Row)>
</Col>
`;
